container/gcloud_bazel_container/Dockerfile.builder

```diff
@@ -0,0 +1,62 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
```

Comment:
As discussed offline, lets try to remove this dockerfile altogether and install the bazel version we need (+gcloud, if needed) as separate steps in the cloudbuild.yaml